### DISCUSSION TITLE
New keyboard shortcut to focus on the current webpage from the address bar

### DISCUSSION THREAD

**Discussion by KorigamiK** [zen-browser/desktop#2265] [Ideas]
there exists no other shortcut or a way to focus into the current contents of the window from the address bar without using your mouse.

A configurable keybind for this would be greatly appreciated!

This is the issue that haunts every day
https://superuser.com/questions/1260644/how-to-move-focus-back-to-the-webpage-in-firefox

**Comment by KorigamiK** [comment]
To be honest Shift + F6 works but is so inaccessible on my keyboard that I would appreciate it if we could configure it to be something else

**Reply by Kamyil** [reply]
+1 Absolutely. I would love to have an option to modify this waaaay waay too inaccessible hotkey

**Comment by dreytko** [comment]
@KorigamiK went to the issues for the same reason but decided to investigate it one more time before writing.
`Alt`+`D`, `Ctrl`+`L` and `F6` (no need to press `Shift`) are Firefox default combos for that (see [Keyboard shortcuts](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly), `Focus Address Bar` option).  
One odd behaviour - `F6` focuses on the address bar, but it doesn't start floating until you type. But `Alt`+`D` and `Ctrl`+`L` are floating from the start.
Still think it's a good feature to have, but for me, `Alt`+`D` feels well placed as-is to remove the configurability need completely. Maybe it will work for you as well.

**Reply by Kamyil** [reply]
Glad it's working for Linux/Windows. But Mac doesn't have `Alt` unfortunately :/ 
It has `Option` and Firefox did not translate `Alt+D` to Mac's `Option+D`, therefore we're still lacking an convenient way to focus URL bar.

**Reply by dreytko** [reply]
Well, that sucks 🙂 
What about `Cmd+L`?
I feel you, but still - even `F6`, when you don't need to hold `Shift` for it, is also not the worst option, right? At least until this feature is done.

**Reply by Kamyil** [reply]
not bad, but it could be better :D

**Comment by KorigamiK** [comment]
Since this issue is active again, I’d like to share how I incorporate a fix for the same:

- for me pressing f6 is not an option and it’s too inaccessible. 
- the easiest way to unfocus the address bar is to press ctrl-h twice. 
- this opens and closes the history tab and puts the focus back to the page
- But it’s still inconvenient because the layout shifts are too aggressive and makes it very jarring. 
- the way I fix this is to open the Firefox sidebar as a floating panel through this css that I wrote
https://github.com/KorigamiK/zen-conf/blob/main/chrome/userChrome.css

Side note:
- you can use a Userscript for fx-autoconfig which can map something like ctrl-shift-l to toggle the sidebar achieving the same result in less keystrokes
